.github/actions/buildbook/action.yml

```diff
@@ -0,0 +1,48 @@
+name: 'Build Jupyterbook'
+description: 'Build the Jupyterbook with desired run features'
+
+inputs:
+  jb-cache:
+    description: “Set up the Jupyterbook Cache (boolean)”
+    required: true
+  publish-to-gh:
+    description: “Publish to GitHub Pages (boolean)”
+    required: true
+  jb-save:
+    description: "Save the Jupyterbook Build (boolean)"
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Setup JupyterBook Cache
+      if: ${{inputs.jb-cache}} == true
+      uses: actions/cache@v2
+      # NOTE: change key to "jupyterbook-N+1" to force rebuilding cache
```

Comment:
Is that a current comment?
I need some help to interpret this. Currently it is not obvious where the key is being set

Reply:
Ahh .. now I remember.

It refers to here:
https://github.com/snowex-hackweek/website/blob/ffd063b9cfba8f4c92642466544d147e776db9ae/.github/workflows/deploy.yaml#L25

I suggest starting with a `jupyterbook-0` key value below to make this a bit more obvious

Reply:
Changed